lib/templates/simple/simple_pubspec_yaml.template.dart

```diff
@@ -0,0 +1,28 @@
+String data({
+  required String name,
+  required String description,
+  required String version,
+  required String flameVersion,
+}) =>
```

Comment:
not perfect but it's what we can do

Reply:
btw I stole this idea from how [dcli](https://bsutton.gitbook.io/dcli/dcli-api/assets) solves this issue